File: programacion/3_Estructuras_de_Datos/2_Ejercicios_Listas_Tuplas/Ejercicio_3_1_15.py
import random

def generar_apuesta_primitiva():
    
    numeros = random.sample(range(1, 50), 6)

    numeros.sort()

    reintegro = random.randint(0, 9)
    
    print("--- TU COMBINACIÓN DE LA PRIMITIVA ---")
    print(f"Números: {numeros}")
    print(f"Reintegro: {reintegro}")
    print("--------------------------------------")


generar_apuesta_primitiva()

File: programacion/4_POO/1_KotlinBasico_EEDD/src/Ejercicio3115.kt
import java.util.Random

fun main() {
    val random = Random()
    val numeros = ArrayList<Int>()

    while (numeros.size < 6) {
        val numAleatorio = random.nextInt(49) + 1

        if (numAleatorio !in numeros) {
            numeros.add(numAleatorio)
        }
    }

    numeros.sort()

    val reintegro = random.nextInt(10)

    println("--- TU COMBINACIÓN GANADORA ---")
    print("Números: ")
    for (i in 0 until numeros.size) {
        print(numeros[i])

        if (i < numeros.size - 1) {
            print(", ")
        }
    }

    println("\nReintegro: $reintegro")
    println("-------------------------------")
}